### DISCUSSION TITLE
Kata5 Ejercicio 1 - Usando funcion round para practicar.

### DISCUSSION THREAD

**Discussion by Eholguin05** [LaunchX-InnovaccionVirtual/CursoIntroPython#122] [Show and tell]
En el ejercicio 1 de la Kata 5, el segundo valor me salió con decimales, así que investigue un poco y utilice la función 'round' para redondear. Espero les sea de su interés y ayuda.

first_planet = 149597870
second_planet = 778547200

distance_km = second_planet - first_planet
print(distance_km)

distance_mi = distance_km * 0.621
round (distance_mi)

**Comment by GloriaCoralCerecedo** [comment]
Hice lo mismo, también creo que utilice absoluto, porque me salían negativos, debido a que hice la reta alrevez, el primer planeta con el segundo

**Comment by doguedogue** [comment]
Exacto, para quitar decimales puedes:
- redondear,
- convertir a entero, 
- redondear hacia arriba ,
- redondear hacia abajo,
- etc.

**Comment by Joselts** [comment]
Hola, yo tengo este problema, me sale este error y ya le intente de muchas formas y ninguna me funciona. 
![image](https://user-images.githubusercontent.com/99304151/154191056-318fb7cf-ab39-4a2e-89ab-7512108981da.png)

**Reply by Ingrzs** [reply]
Tienes separado print (Distancia)

Fijate debajo tambien tienes print y no lo tienes separado, checa bien eso de los espacios.

**Reply by EnriqueIbarraM** [reply]
> Tienes separado print (Distancia)
> 
> Fijate debajo tambien tienes print y no lo tienes separado, checa bien eso de los espacios.

Acabo de replicar su código y no afecta la separación de la función con los paréntesis.
@Joselts El código funcionó sin problema.

**Reply by Ingrzs** [reply]
Que raro si lo replicaste también debio salirle bien, o quizas la razón es que el valor de input no lo esta conviertiendo a tipo int , y como esta restando, por eso arroja error...

**Reply by OscarIslasH** [reply]
En donde pusiste (Millasfloat) Va el valor de las millas, osea 0.621 quedaria como: distance_mi = distance_km * 0.621 
No hace falta que pongas los parentesis, lo que si, es que te falto solo uno al final del print : print(abs(distance_mi))

**Comment by JorgeCanto3** [comment]
De donde sale el .621?

**Comment by EnriqueIbarraM** [comment]
Es el equivalente, en millas, a 1 Km.
________________________________
De: JorgeCanto3 ***@***.***>
Enviado: martes, 15 de febrero de 2022 10:16 p. m.
Para: LaunchX-InnovaccionVirtual/CursoIntroPython
Cc: EnriqueIbarraM; Comment
Asunto: Re: [LaunchX-InnovaccionVirtual/CursoIntroPython] Kata5 Ejercicio 1 - Usando funcion round para practicar. (Discussion #122)


De donde sale el .621?

—
Reply to this email directly, view it on GitHub<https://github.com/LaunchX-InnovaccionVirtual/CursoIntroPython/discussions/122#discussioncomment-2185864>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AXVPUMFK7VYZZWCGSJO6E23U3MQJ3ANCNFSM5ODHBPCQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Reply by JorgeCanto3** [reply]
Pero porque dice que lo multipliquemos por 1.6..., es un error?

**Reply by EnriqueIbarraM** [reply]
Sí, estaba mal.
Pero ya lo corrigió Fernanda.

**Comment by Joselts** [comment]
Muchas gracias, no se que paso pero lo copie en un nuevo notebook y el código corrió sin problemas, muchas gracias por la ayuda.
![image](https://user-images.githubusercontent.com/99304151/154288677-a0fca456-5f62-424b-a7f0-82796175d2b8.png)
